packages/deprecation-crawler/tests/index.test.ts

```diff
@@ -76,7 +76,7 @@ test('sandbox', async () => {
       (d) => d.group === 'multiple-string-patterns-at-once'
     )
   ).toHaveLength(0);
-  // BUG: this has to have 0 hits
+  // @TODO BUG: this has to have 0 hits
```

Comment:
Should we do something here?

Reply:
Yes, probably 😅
There's something wrong with the regex I think.

Reply:
What should I check? Give me a request and I solve it.

Reply:
See #94